jobs/integr8ly/pds-testing-executor.yaml

```diff
@@ -44,6 +44,57 @@ This needs to be changed to 'false' for Integreatly Workshop since it uses valid
             name: TEST_SUITES_BRANCH
             default: master
             description: "Branch of a repository of Integreatly test suites"
+        - string:
+            name: EVALS_USERNAME
+            default: 'evals11@example.com'
+            description: 'Evals account email address used in test suites.'    
+        - string:
+            name: EVALS_PASSWORD
+            default: 'Password1'
+            description: 'Evals user password used in test suites'
+        - string:
+            name: ADMIN_USERNAME
+            default: 'admin@example.com'
+            description: 'Admin sername to login to Integreatly cluster.'
+        - string:
+            name: ADMIN_PASSWORD
+            default: 'Password1'
+            description: 'Admin password to login to Integreatly cluster.'
+        - string:
+            name: CUSTOMER_ADMIN_USERNAME
+            default: 'customer-admin@example.com'
+            description: 'customer-admin username to login to Integreatly cluster.'
+        - string:
+            name: CUSTOMER_ADMIN_PASSWORD
+            default: 'Password1'
+            description: 'Password to login to Integreatly cluster.'
+        - string:
+            name: RELEASE_VERSION
+            default: 'master'
+            description: 'This optional parameter allows to test particular release version. If not provided, master will be used'
+        - bool:
+            name: CLEAN_RESOURCES
+            default: true
+            description: 'depending on whether the resources should be cleaned after the successful execution of certain tests'
+        - string:
+            name: MANIFEST_VERSION
+            default: 'master'
+            description: 'Provide release version when testing particular release (smoke tests)'
+        - bool:
+            name: TESTING_MASTER
+            default: true
+            description: 'Indicates whether to check master tags or compare versions from manifest file (smoke tests)'   
+        - string:
+            name: WALKTHROUGHS_VERSION
+            default: ''
+            description: 'Provide walkthroughs release version. Empty value means the master version is expected (smoke tests)'
+        - string:
+            name: SSO_URL
```

Comment:
There is neither CLUSTER_URL or WEBAPP_URL, these two are constructed from YOURCITY and NAMESPACE_PREFIX parameters. The SSO_URL can be handled this way as well ( SSO_URL equals to https://sso-${NAMESPACE_PREFIX}sso.apps.${YOURCITY}.openshiftworkshop.com).

For the sake of consistency, I would prefer handling the SSO_URL the same way.

Reply:
@trepel I have addressed your comment. Thanks for the review.